tests/module/utils/test_bezier.py

```diff
@@ -95,3 +97,73 @@ def test_subdivide_bezier() -> None:
                 subdivide_bezier(points, n_divisions),
                 subdivision_matrix @ points,
             )
+
+
+def test_get_smooth_cubic_bezier_handle_points() -> None:
+    """Test that :func:`get_smooth_cubic_bezier_handle_points` returns the
```

Comment:
```suggestion
    """Test that :func:`.get_smooth_cubic_bezier_handle_points` returns the
```
Although I don't think tests are documented so it shouldn't make a difference :)